examples/certificate.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "Deploy Postgresql server with ssl support,
+         key and certificate is provided by certificate system role"
+  hosts: localhost
+  vars:
+    pg_hba_conf:
+      - type: hostssl
+        database: all
+        user: all
+        auth_method: md5
+        address: '127.0.0.1/32'
+    ssl_enable: yes
```

Comment:
Shouldn't be `ssl_enable` indended to be under `pg_hba_conf`? Or maybe it should be `postgresql_ssl_enable`?